setup.py

```diff
@@ -6,7 +6,20 @@
 from setuptools import Extension, setup
 from setuptools.command.bdist_rpm import bdist_rpm as _bdist_rpm
 from setuptools.command.build_ext import build_ext
-from setuptools.errors import CCompilerError, ExecError, PlatformError
+
+
+try:
+    from setuptools.errors import CCompilerError, ExecError, PlatformError
+except ImportError:
+    # RTD workaround until it ships setuptools>=v59.0.0
```

Comment:
Could you explain what this change for? The current version of setuptools is 60.2.0